go.sum

```diff
@@ -300,6 +300,8 @@ github.com/redhat-appstudio/integration-service v0.0.0-20231025084434-b3f521c408
 github.com/redhat-appstudio/integration-service v0.0.0-20231025084434-b3f521c408d1/go.mod h1:bh8uJ4whJ92ef5PXH6Rfm4F8tRXkDAlSYR7Pqp1LLT4=
 github.com/redhat-appstudio/operator-toolkit v0.0.0-20230913085326-6c5e9d368a6a h1:pwTvkRzRF6zyLW1Bb4vEuqaNXlGJOnBtt9n5gNp0/r4=
 github.com/redhat-appstudio/operator-toolkit v0.0.0-20230913085326-6c5e9d368a6a/go.mod h1:7cX2+4KGZLJ4Yoj+1v0iV5hkCGBzbSd9wkNJQjCdDJs=
+github.com/redhat-appstudio/release-service v0.0.0-20230511145849-bde1cdcbb60b h1:96jgqIR8Otx2vrLZscvNo3gFlX2d9P/mkbx5Trkm8tQ=
```

Comment:
This doesn't look right

Reply:
When I ran `go mod tidy,` the following lines were added.
```
github.com/redhat-appstudio/release-service v0.0.0-20230511145849-bde1cdcbb60b h1:96jgqIR8Otx2vrLZscvNo3gFlX2d9P/mkbx5Trkm8tQ=
```

I am not sure when the new package will be available for konflux-ci/release-service
like this https://pkg.go.dev/github.com/redhat-appstudio/release-service/api/v1alpha1 

Once we merge this PR? with the updated module name?

Reply:
Okay, can you put investigating once this PR merges if we can remove this line and if not, say why on your todo list?

Reply:
Definitely!